ui/components/app/detected-token/detected-token-values/detected-token-values.js

```diff
@@ -59,7 +56,7 @@ const DetectedTokenValues = ({
         </Text>
       </Box>
       <Box className="detected-token-values__checkbox">
```

Comment:
Can the styles be removed in favour of the style utility props? marginLeft="auto"